parameters.satisfy.yml

```diff
@@ -0,0 +1,6 @@
+parameters:
+    secret: "QrsEr!U4=qBSJH-3&AdTaZp!4R=NX*UD"
+    satis_filename: "/app/config.json"
+    satis_log_path: "%kernel.root_dir%/../var/satis"
+    admin.auth: false
+    admin.users: ~
```

Comment:
Need more doc in README please.

Reply:
I don't think so. Becouse that is come with the change to ["Symfony 3.3 micro kernel"](https://github.com/ludofleury/satisfy/commit/1d8dcd55f7631565704917eafa30110c8a008040) and it will be install normal if composer start in interaction mode. But we start composer in --no-interaction mode.

Reply:
so we make at manuel